TestResultSummaryService/parsers/BenchmarkMetricRouter.js

```diff
@@ -18,6 +18,9 @@ const BenchmarkMetricRouter = {
     },
     "SPECjbb2015": {
         "multi_2grp_gencon": "SPECjbb2015"
+    },
+    "AcmeAir_Nodejs": {
+        "linux-s390x" : "AcmeAir_Nodejs"
```

Comment:
Can you please give a more meaningful name to the variant? Something generic like "default" or "throughput" that is independent of the platform `linux-s390x`. If you decide to run this benchmark on other platforms in future, then this variant name won't make much sense.

Reply:
Thank you for the suggestion. changed it to 1grp.